int8 index
float32 min_range
float32 max_range
float32 distance 
float32 humidity
string warning


